dotCMS/src/main/java/com/dotmarketing/portlets/templates/action/EditTemplateAction.java

```diff
@@ -319,20 +330,34 @@ else if ((cmd != null) && cmd.equals(com.dotmarketing.util.Constants.FULL_DELETE
 				String [] inodes = req.getParameterValues("publishInode");
 
 				int errorCount =0;
+				SessionDialogMessage errors = new SessionDialogMessage(
+						LanguageUtil.get(user, "Delete-Template"),
+						LanguageUtil.get(user, "message.template.full_delete.error"),
+						LanguageUtil.get(user.getLocale(), "message.template.dependencies.top", 10) + "<br>" +
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 10 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS109)